templates/trivy/trivy-sts.yaml

```
@@ -1,3 +1,4 @@
+{{ if .Values.trivy.enabled }
```

I think you missed a curly bracket here ;)
```suggestion
{{- if .Values.trivy.enabled }}
```

---

Oh my, sorry .. :) 👍